Separate stress tests out and run them on node 1

ararslan opened the pull request:
This ensures that they don't interfere with other tests. Suggested by @vtjnash in https://github.com/JuliaLang/julia/pull/28201#issuecomment-407520112. Let me know if any other tests belong here as well.

ararslan committed:
Separate stress tests out and run them on node 1

This ensures that they don't interfere with other tests.

vtjnash reviewed (approved):
Perhaps `test_13559` and `test_22566` and the test for `jl_exit_on_sigint` also, if you're looking for more. I assume others will move or be added over time.

iblis17 commented:
`test_13559` and `test_22566` doesn't cause huge workload on pipe. So I hope the `file` testsuit won't freeze FreeBSD CI builds after this PR merged.

ararslan committed:
Move tests for #13559, #22566, and jl_exit_on_sigint to stress.jl

iblis17 commented on test/stress.jl:
```diff
@@ -0,0 +1,107 @@
+
+# issue #22566
+# issue #24037 (disabling on FreeBSD)
+if !Sys.iswindows() && !(Sys.isbsd() && !Sys.isapple())
```
Good to enable it on FreeBSD?

ararslan replied:
I don't know, is it? 😄 If you reenable locally it can you reproducing the freezing it was causing before?

iblis17 replied:
> If you reenable locally it can you reproducing the freezing it was causing before?

I reenabled it.... seems fine now.

ararslan replied:
Great! Thanks for checking, I'll reenable it here.

ararslan committed:
Reenable the test for #22566 on FreeBSD

It was disabled in #24037 but reportedly seems to work okay when
isolated into this separate, node 1 only test group.

iblis17 commented:
I want to build at least 3 times to make sure no freezing again.

- [ ] https://freebsdci.julialang.org/#/builders/1/builds/11595
- [ ] TBD
- [ ] TBD

ararslan commented:
There were test failures on Travis Linux (both 32- and 64-bit) related to Sockets not being able to connect, so I've restarted those builds to be safe. IIRC we don't run the Sockets tests at all on Circle.

ararslan commented:
All good now. I'll go ahead and merge.
